seaborn/categorical.py

```diff
@@ -2870,3 +3098,193 @@ def factorplot(x=None, y=None, hue=None, data=None, row=None, col=None,
         <seaborn.axisgrid.FacetGrid object at 0x...>
 
     """).format(**_categorical_docs)
+
+def lettervalueplot(x=None, y=None, hue=None, data=None, order=None, hue_order=None,
+            orient=None, color=None, palette=None, saturation=None,
+            width=None, k_depth=None, linewidth=None, box_widths=None,
+            ax=None, **kwargs):
+
+    # Try to handle broken backwards-compatability
```

Comment:
You don't need this section, it's only relevant to handle the changes from < 0.6 in `boxplot` and `violinplot`, but nobody would have old `lvplot` code using these parameters.